src/main/resources/openapi/tasks.yaml

```diff
@@ -114,11 +114,11 @@ components:
   schemas:
     AutomatedTaskPatch:
       type: object
-      required:
-        - is_active
       properties:
         is_active:
           $ref: '#/components/schemas/IsActive'
+        batch_size:
+          type: integer
```

Comment:
author note:
To do things the perfect way we should now validate that at least one of the patch attributes are not null.
The Open Api documentation says that Open Api can enforce that by using `minProperties: 1` but unfortunately it won't work.
I have therefore decided not to enforce this because:

- This is not a public API, and the Portal won't allow sending all `null` patch attributes
- I'd rather avoid overcomplicating the code by adding manual validation logic for this when not really needed
- Even if the endpoint was called with all `null` attributes, the implementation would ignore them and won't change the state of the task
- other PATCH methods in our APIs don't enforce this either